src/draft/NSG/Agent/LegalEntity/NonListedCompany/BeneficialOwners.py

```diff
@@ -0,0 +1,98 @@
+from enum import Enum
+from typing import List
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class ShareSeriesClass(str, Enum):
+    A = "A"
+    B = "B"
+    C = "C"
+    D = "D"
+    E = "E"
+
+
+class Ownerships(CamelCaseModel):
```

Comment:
I think this comment is mainly for @hsuomi, but please also @omarudolley comment:

I'm a bit wondering if this name of the class is correct and logical. Each shareholder has one or more of these objects (depending on if the person has shares in multiple share series or just one) and that's why we have a list of these. Of course each of these objects typically contain X amount of shares (the `quantity`), so that way each of these does correspond to ownership of multiple shares in a share series class.

One of my thought is that should this class be in plural or not (`Ownerships` vs `Ownership`), as each of these represent the ownership in one particular share series class.

But I'm also wondering if each of these is just an `ownership`/`ownerships`, as each of these only correspond to ownership(s) of one class of shares. I'm thus a bit thinking if this class should be named something like `ShareSeriesOwnership`/`ShareSeriesClassOwnership`.

Another option could be to call this class just `Shares`, so your ownership consists of a list of shares. (And since each of these has a quantity, I think it makes sense to have it in plural).

---

I decided to still check the dictionary a bit regarding the word `ownership` and it lists that word as uncountable (see https://www.oxfordlearnersdictionaries.com/definition/english/ownership). Thus I think the list of these objects (in the `Shareholders`) should be called `ownership`, not `ownerships` and if we stick with the class name as something ending in ownership it should not have an `s` at the end.

Reply:
Thanks for raising! True that ownership is uncountable so there is a bit of conflict now in here indeed. I agree we should have it raised in the form of Ownership and I like the proposal actually adding the word share there. Now when I checked "Share Ownership" is at least what some legal companies use and Let's use that for now. I updated the spreadsheet accrdingly and highlighted with pink. @omarudolley  Please remember to check the updated definition as well.